virttest/utils_test/qemu/__init__.py

```diff
@@ -735,6 +753,9 @@ def check_memory(self, vm=None, wait_time=0):
         #  let system perceive it.
         time.sleep(wait_time)
         guest_mem_size = super(MemoryHotplugTest, self).get_guest_total_mem(vm)
+        if self.os_type == "linux":
+            crashkernel_size = super(MemoryHotplugTest, self).get_guest_crashkernel_size(vm)
+            guest_mem_size += crashkernel_size
         vm_mem_size = self.get_vm_mem(vm)
         if abs(guest_mem_size - vm_mem_size) > vm_mem_size * threshold:
```

Comment:
Although we improved the accuracy of calculating the memory size under the specific testing scenario, we used the previous way of computing this part by muting the `threshold` that the user will give it.
So there is one concern from my side, whether this change will affect the accuracy of test cases for other platforms.
IMO, we should run the regression testing for this change.
Hi @zhenyzha 
Would you please help to `@` the other platform's reviewers?
Thanks.

Reply:
Yes, I agree.

@mcasquer @MiriamDeng @fbq815 Could you pls help review when you are free. thanks.

Reply:
@zhenyzha executed mem_hotplug test loop after applying this patch, all the hotplug cases passed as expected
```bash
 (001/141) Host_RHEL.m9.u3.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.3.0.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: STARTED
 (001/141) Host_RHEL.m9.u3.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.3.0.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (1133.38 s)
...
 (141/141) Host_RHEL.m9.u3.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.3.0.x86_64.io-github-autotest-qemu.hotplug_mem_negative.invalid_device_addr.q35: STARTED
 (141/141) Host_RHEL.m9.u3.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.3.0.x86_64.io-github-autotest-qemu.hotplug_mem_negative.invalid_device_addr.q35: PASS (119.99 s)
RESULTS    : PASS 139 | ERROR 0 | FAIL 2 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
```

Reply:
currently all memory-hotplug test cases uses NUMA and it's not supported on s390x. I think we could skip on s390x for this time

Reply:
Hi @MiriamDeng 
As the changes of hugepage in the avocado-vt framework and avoiding hitting the regression, could you please help to run the `slof_hugepage` case?
Many thanks.

Reply:
Test pass
 (09/15) Host_RHEL.m8.u9.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.3.0.ppc64le.io-github-autotest-qemu.slof_hugepage.with_backing_file_hotplug: STARTED
 (09/15) Host_RHEL.m8.u9.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.3.0.ppc64le.io-github-autotest-qemu.slof_hugepage.with_backing_file_hotplug: PASS (224.50 s)
 (10/15) Host_RHEL.m8.u9.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.3.0.ppc64le.io-github-autotest-qemu.slof_hugepage.without_backing_file_hotplug: STARTED
 (10/15) Host_RHEL.m8.u9.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.3.0.ppc64le.io-github-autotest-qemu.slof_hugepage.without_backing_file_hotplug: PASS (165.35 s)